docs/content/docs/install/manual_demo/_index.md

```diff
@@ -71,10 +71,10 @@ osm namespace add bookstore bookbuyer bookthief bookwarehouse
 Deploy `bookbuyer`, `bookthief`, `bookstore`, `bookwarehouse` applications:
```

Comment:
can you add a note somewhere that all these yamls should be applied from the root of the repo?